### ISSUE TITLE
Accessible sass themes

### ISSUE THREAD

**Issue by jivanova** [telerik/kendo-themes#2468]
**:x: Failure:** Elements must have sufficient color contrast 
_Disabled elements is not required to pass the color contrast criteria_

- [ ] Ensure contrast in Default theme and Bootstrap theme
   - contrast between `light` and `dark`
   - contrast between `primary color` and `light color` for light themes and `primary color` and `dark color` for dark themes
- [ ] Ensure contrast of the themecolors
- [ ] Adopt contrast-wcag function in components 
_*might need to research the bootstrap contrast definitions_

Attached report from axe testing for Material theme of the current visual tests:

[Material_AllErrors_Filtered.xlsx](https://github.com/telerik/kendo-themes/files/6447187/Material_AllErrors_Filtered.xlsx)

Spike of axe testing visual theme tests could be found [here ](https://github.com/jivanova/axe-themes) and used for reference.

**Comment by joneff** [comment]
Contrast between primary and light or primary and dark cannot be ensured. What can be ensured is:

* for dark themes: contrast between default text color and dark
* for light themes: contrast between default text color and light

**Comment by veselints** [comment]
I am not quite sure that the following will be useful. Some time ago I have performed a small review of the color contrast for all SASS themes (Default, Bootstrap, Material). The conclusion was that the Material theme exhibits the smallest number of contrast issues. The Bootstrap and Default had much more issues compared to the Material.

**Comment by jivanova** [comment]
closing this with the release of Ocean blue swatch
